Automattic/wp-calypso
# Remove reader/post-options

The component `reader/post-options` originally provided the ellipsis menu on Reader post cards (containing functions like 'Follow' and 'Edit').

The component was migrated to `blocks/reader-post-options-menu` during the Reader refresh, and the old version in `reader/post-options` is now redundant.

### To test

On a Reader post card, click the ellipsis menu and ensure the menu functions as expected.

<img width="299" alt="screen shot 2017-01-05 at 16 02 26" src="https://cloud.githubusercontent.com/assets/17325/21669509/682a15e6-d360-11e6-97a5-f80c9d748ad1.png">
